plugins/modules/idrac_reset.py

```diff
@@ -3,64 +3,147 @@
 
 #
 # Dell OpenManage Ansible Modules
-# Version 7.1.0
-# Copyright (C) 2018-2022 Dell Inc. or its subsidiaries. All Rights Reserved.
+# Version 9.2.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
 
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 #
 
-
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-DOCUMENTATION = """
+DOCUMENTATION = r'''
 ---
 module: idrac_reset
-short_description: Reset iDRAC
-version_added: "2.1.0"
+short_description: Factory reset the iDRACs
+version_added: "9.2.0"
```

Comment:
Keep this as 2.1.0, it was added at that releases

Reply:
Done